luminoth/detect.py

```diff
@@ -0,0 +1,40 @@
+import click
+import easydict
+import tensorflow as tf
+import os
+import yaml
+
+from luminoth.utils.predict import get_prediction
+from PIL import Image, ImageDraw
+
+
+@click.command(help='Detects objects in a image.')
+@click.argument('image-path')
+@click.option('--config-file')
+@click.option('--model-name', default='fasterrcnn')
+def detect(image_path, config_file, model_name):
```

Comment:
This should primarily output a JSON file with the predictions. Think that somebody will want to use this programmatically, and giving this person the images will not do any good (unless for debugging).

There should be an argument (disabled by default) that lets you also output the image with the predictions drawn on top, but this should be optional.

Reply:
Optionally, this could also be a separate command that takes an input image, and predictions (in json format) and draws whatever it has to draw over the image.

To discuss.